molecule/default/playbook.yml

```diff
@@ -0,0 +1,36 @@
+---
+- hosts: all
+
+  pre_tasks:
+
+    - name: Create unprivileged test user
+      user:
+        name: test
+        uid: 12345
+
+    - name: Create directories
+      file:
+        path: "{{ item }}"
+        state: directory
+        owner: test
+      with_items:
+        - /srv/share
+        - /mnt/ro
+        - /mnt/rw
+
+  roles:
+
+    - role: ome.nfs_share
+      nfs_shares:
+        /srv/share:
+          - host: "*"
+            options: "rw,fsid=0"
+
+    - role: ansible-role-nfs-mount
+      nfs_share_mounts:
+        - path: /mnt/rw
+          location: localhost:/share
```

Comment:
Should this be `/srv/share` ?

Reply:
It's complicated due to the workarounds needed to get NFS shares working in Docker (`fsid=0`). This is fixed in the last commit.